libraries/AP_Landing/AP_Landing.cpp

```diff
@@ -410,6 +598,8 @@ bool AP_Landing::request_go_around(void)
     case TYPE_STANDARD_GLIDE_SLOPE:
         success = type_slope_request_go_around();
         break;
+    case TYPE_DEEPSTALL:
+        return type_deepstall_request_go_around();
```

Comment:
why are you intentionally skipping the log event here? In log you don't even log! I broke up the logging like this to make it easier on you. Maybe we should re-think the logging infrastructure?